knotx-core/knotx-template-engine/src/main/java/com/cognifide/knotx/engine/service/ServiceEngine.java

```diff
@@ -3,29 +3,27 @@
  *
  * Copyright (C) 2016 Cognifide Limited
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
```

Comment:
Please provide proper header formatting (as in other files).
This class is also part of `Templating Engine Verticle` which should be mentioned here.